server/services/damsService/queries.js

```diff
@@ -134,7 +134,7 @@ function generateGetAssetsByFileNameQuery(ensembleIndex) {
           {
             operator: "and",
             exact: {
-              attribute: "API Ensemble Index",
+              attribute: "API Ensemble Index (TMS)",
```

Comment:
Apparently sending an incorrect attribute name will break the NetX query request entirely rather than them return an empty list